app/src/main/java/org/jellyfin/mobile/fragment/WebViewFragment.kt

```diff
@@ -158,15 +158,14 @@ class WebViewFragment : Fragment(), NativePlayerHost {
                 val url = request.url
                 val path = url.path?.lowercase(Locale.ROOT) ?: return null
                 return when {
-                    !connected && path.endsWith(Constants.ANY_BUNDLE_PATH) -> {
+                    path.matches(Constants.MAIN_BUNDLE_PATH_REGEX) && url.query.isNullOrEmpty() -> {
```

Comment:
I'd add a special query parameter in the injection script the check here. I can imagine jellyfin-web adding a cache buster at some point which would break this detection

Reply:
> I'd still put it in the head tbh

But why?

Reply:
> I'd add a special query parameter in the injection script the check here. I can imagine jellyfin-web adding a cache buster at some point which would break this detection

That'd also break appending the timestamp in the JS..

Reply:
Also, isn't the hash inside the filename already a cache-buster?

Reply:
> I'd add a special query parameter in the injection script the check here. I can imagine jellyfin-web adding a cache buster at some point which would break this detection

Don't worry, that will not happen because it is already [included](https://github.com/jellyfin/jellyfin-web/blob/6fc729f59351ef3bc313eea0e1da0dfa3236233a/webpack.common.js#L77) in the filename.

> v10.7.5 `main.4b4b736dbefe7c3a90ec.bundle.js`
> v10.7.6 `main.0f183787a2f78055e856.bundle.js`

> Also, isn't the hash inside the filename already a cache-buster?

Yep

Reply:
Did some more tweaks, should be good now.

Reply:
The fact that the hash is included in the filename doesn't mean someone can add a query string to the file in the future. I think it's important to create an implementation that is future-proof.

Reply:
Yep, that's why I changed it. I think this one can be resolved.